VMware.CloudFoundation.PasswordManagement.psm1

```diff
@@ -2401,8 +2401,28 @@ Function Update-SddcManagerPasswordComplexity {
                             if($existingConfiguration.'Min Classes') {
                                 $chkExistingConfig = $chkExistingConfig -or $existingConfiguration.'Min Classes' -ne $minClass
                             }
-                            if ($chkExistingConfig) {
-                                Set-LocalPasswordComplexity -vmName ($server.Split("."))[-0] -guestUser root -guestPassword $rootPass -minLength $minLength -uppercase $minUppercase -lowercase $minLowercase -numerical $minNumerical -special $minSpecial -unique $minUnique -class $minClass -sequence $maxSequence -history $history -retry $maxRetry | Out-Null
+                            $sddcManagerVersion = Get-VCFManager -version
+                            if (($sddcManagerVersion.split(".")[0] -eq 5 -and $sddcManagerVersion.split(".")[1] -ge 1) -and ($existingConfiguration.'Min Length' -eq $null -or $existingConfiguration.'Min Lowercase' -eq $null -or $existingConfiguration.'Min Uppercase' -eq $null -or $existingConfiguration.'Min Numerical' -eq $null -or $existingConfiguration.'Min Special' -eq $null -or $existingConfiguration.'Min Unique' -eq $null -or  $existingConfiguration.'History' -eq $null -or $existingConfiguration.'Max Retries' -eq $null -or $existingConfiguration.'Max Sequence' -eq $null -or $existingConfiguration.'Min Classes' -eq $null)) {
```

Comment:
Shouldn't this be `-ge`?

Otherwise, this must be revisited for any release after 5.1.